javascript.js

```diff
@@ -23,3 +25,270 @@ document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click'
+};
+const project1 = {
+  title: 'Website Portfolio', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-2.png',
+};
+const project2 = {
+  title: 'Data Dashboard', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-3.png',
+};
+const project3 = {
+  title: 'Brand website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-4.png',
+};
+const project4 = {
+  title: 'Shopping Website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-2.png',
+};
+const project5 = {
+  title: 'School Website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-3.png',
+};
+const project6 = {
+  title: 'eLearning Website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-4.png',
+};
+
+// add objects to array
+arrayElements.push(project1);
+arrayElements.push(project2);
+arrayElements.push(project3);
+arrayElements.push(project4);
+arrayElements.push(project5);
+arrayElements.push(project6);
+
+function addBlur() {
+  document.querySelector('header').setAttribute('style', 'filter: blur(5px)');
+  document.querySelector('main').setAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#Portfolio').setAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#About').setAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#Contact').setAttribute('style', 'filter: blur(5px)');
+  document.querySelector('footer').setAttribute('style', 'filter: blur(5px)');
+}
+
+function removeBlur() {
+  document.querySelector('header').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('main').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#Portfolio').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#About').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('#Contact').removeAttribute('style', 'filter: blur(5px)');
+  document.querySelector('footer').removeAttribute('style', 'filter: blur(5px)');
+}
+
+// Dynamic content for main featured project card
+const main_project_container = document.querySelector('.featured_project');
+const main_card_content = document.querySelector('.main_card');
```

Comment:
- [ ] it would be better to use the right background color as in the Figma design in order to match it 
- Yours 
<img width="851" alt="Screenshot 2023-03-09 at 12 16 42" src="https://user-images.githubusercontent.com/61790587/224008905-11b119d0-310a-4b2d-83b2-fda069515af3.png">
Design 
<img width="867" alt="Screenshot 2023-03-09 at 12 15 48" src="https://user-images.githubusercontent.com/61790587/224008966-86c0b883-c876-406b-b622-10961d41aa6e.png">